src/Language/PureScript/TypeClassDictionaries.hs

```diff
@@ -8,10 +8,10 @@ import Language.PureScript.Types
 -- |
 -- Data representing a type class dictionary which is in scope
 --
-data TypeClassDictionaryInScope
+data TypeClassDictionaryInScope n
   = TypeClassDictionaryInScope {
     -- | The identifier with which the dictionary can be accessed at runtime
-      tcdName :: Qualified Ident
+      tcdName :: n
```

Comment:
I think I tried something like that previously (think it was `Either Expr (Qualified Ident)`) but it caused a module dependency cycle.  I'll have another look later today.